### PR TITLE
Minor cleanup patches

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +101,7 @@ This project will hence provide a CLI tool and a Rust library which speaks the D
 An important aspect of this is that the system will validate the GPG signature of the target OSTree commit, as well as validating the sha256 of the contained objects.
 
 ```
-$ ostree-container pull --repo=/ostree/repo --ref=exampleos/x86_64/stable quay.io/exampleos/exampleos:stable
+$ ostree-ext-cli container import --repo=/ostree/repo quay.io/exampleos/exampleos:stable
```
Comment (kelvinfan001) [inline README.md:104]: Not super familiar with `ostree-ext-cli`, but is a ref not required here anymore?

Comment (cgwalters) [reply README.md:104]: I wrote the whole README.md first as an aspirational thing, and then wrote the code.  The code doesn't current support `--ref` but it totally should.

Comment (cgwalters) [reply README.md:104]: (Just trying to correct README.md to reflect reality today)

File: README.md [hunk 2]
```diff
@@ -49,45 +49,44 @@ This is used by `rpm-ostree ex apply-live`.
...
 an arbitrary container image and just makes it "ostree ready".  Or even
 just dyanamically accepting a container image that has a kernel client side.
 
 This *may* be in scope at some point in the future.
 
-#### ostree-containers and derivation
-
-For an ostree-based OS that is derived from Fedora, 
-`ostree-container build --from=registry.fedoraproject.org/fedora:33` would cause the generated container image to derive from the parent; in particular we de-duplicate content in the ostree commit from the base.
+#### Maybe: ostree-containers and derivation
```
Comment (kelvinfan001) [inline README.md:76]: I wasn't sure if this is meant to be a subsection of "Maybe: Take an arbitrary container and convert it to be OSTree ready". (There's an additional `#`)

**Review by kelvinfan001 — APPROVED** [review]
This looks generally sane to me!

File: README.md (hunk 2 above)
Comment (cgwalters) [reply README.md]: OK I think it's clearer if that stuff lives in issues, not as "Maybe:" in the README.md.

**Pull request merged**